tests/test_compress.py

```diff
@@ -25,3 +26,32 @@ def test_compress(outputs_path, output_files, format, remove_originals):
     elif "tar" in format:
         with tarfile.open(package_path, "r:*") as tf:
             assert len(list(tf))
+
+
+@pytest.mark.parametrize("format", ("zip", "tar", "tar.gz"))
+@pytest.mark.parametrize("remove_originals", (False, True))
+@pytest.mark.parametrize("filter", (("**", 6), ("**/*.jpg", 2)))
+def test_compress_wildcards(tmpdir, format, remove_originals, filter):
```

Comment:
Assuming the numbers mean the number of expected files for each filter, you can do

```suggestion
@pytest.mark.parametrize("filter, expected_files", [("**", 6), ("**/*.jpg", 2)])
def test_compress_wildcards(tmpdir, format, remove_originals, filter, expected_files):
```
to avoid having to do `filter[0]` and `filter[1]` in the test.